app/individualBookPage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+require "../vendor/autoload.php";
+
+if (isset($_GET['id'])) {
+    $conn = new \App\DbConnector();
+    $individualBook = new \App\Book($conn->getDb(), $_GET['id']);
+}
+?>
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="css/lib/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <title>View Book:<?php echo $individualBook->title; ?> </title>
+</head>
+<body>
+
+<div class="stickyFooterExcluder">
+    <?php include "includes/header.php"; ?>
+
+    <div class="container">
+        <div class="row content">
+            <div class="col-xs-5 individualBookImage">
+                <img src="<?php echo $individualBook->image; ?>"/>
```

Comment:
what if individual book is empty? no error handling, u need to display an error that says the book does not exist

Reply:
fixed